test/hash_bindings_test.rb

```diff
@@ -28,6 +28,17 @@ class SongWithRepresenter < ::Song
         assert_equal Representable::Binding::FragmentNotFound, @property.read({}, "song")
       end
 
+      it "will not fail if given an empty hash value and will return FRAGMENT_NOT_FOUND" do
+        not_failed = true
+        result = nil
+        begin
+          result = @property.read(nil, "song")
+        rescue NoMethodError
```

Comment:
@toadle we don't have to consider rescuing `NoMethodError` now as we know it shouldn't occur. If it does, then it must be due to some other bug and should be handled differently.

So in this test, we can just assert on return value of `read`, like in above test.

```ruby
it "..." do
  assert_equal Representable::Binding::FragmentNotFound, @property.read(nil, "song")
end
```

Reply:
@yogeshjain999 here you go

Reply:
@yogeshjain999 Wanna merge?

Reply:
Sorry, almost forgot :)